### DISCUSSION TITLE
Problem with "area"-config

### DISCUSSION THREAD

**Discussion by SimenJensen** [cSharpBastard/mmm-nordic-electrical-spot-prices#3] [Q&A] [answered]
When i use the module with default area-config-settings, or change for example the config.js `area: "FI"` - it works fine, but when i change config.js to `area: "NO5"`, it doesn't seem to be working. Any suggestions for why this doesn't work?

My config.js:
`
{
			module: 'mmm-nordic-electrical-spot-prices',
	 		position: 'lower_third',
			config: {
				currency: "NOK",
				area: "NO5",
				title: "Dagens strømpris"
            			}
},
`

**Comment by cSharpBastard** [comment]
I'll take a look

**Comment by cSharpBastard** [comment, marked as answer]
Seems like Nordpool has changed the area codes for Norway. These are available för Norway:
Oslo	
Kr.sand	
Bergen	
Molde	
Tr.heim	
Tromsø

See here: https://www.nordpoolgroup.com/en/Market-data1/#/nordic/table
